docs/setup.md

```diff
@@ -46,7 +84,7 @@ A detailed explanation of the setup script appears below.
 
 ```bash
 # Step 1: Clone this repository and enter the project directory.
-git clone https://github.com/yalesites-org/yalesites-project.git
+git clone git@github.com:yalesites-org/yalesites-project.git
 cd yalesites-project
 
 # Step 2: Ignore the composer.lock file on local dev only.
```

Comment:
"When working on a Yale-managed package" -> "When working on a Yale-managed package such as Atomic or the Component Library Twig"

Reply:
Love that change!  Much more explicit.  Thank you!